src/adapter/src/coord/sequencer/cluster.rs

```diff
@@ -671,30 +676,37 @@ impl Coordinator {
                     name,
                     &compute,
                     new_size,
-                    &mut ops,
+                    &mut create_ops,
                     // AVAILABILITY ZONES hasn't changed, so existing replicas don't need to be
                     // rescheduled.
                     Some(new_availability_zones.as_ref()),
                     *new_disk,
+                    new_replica_start_pending,
                     owner_id,
                     reason.clone(),
                 )?;
                 create_cluster_replicas.push((cluster_id, id))
             }
         }
 
-        let variant = ClusterVariant::Managed(new_config);
-        ops.push(catalog::Op::UpdateClusterConfig {
-            id: cluster_id,
-            name,
-            config: ClusterConfig { variant },
-        });
-
-        self.catalog_transact(session, ops).await?;
+        // we only finalize if we created something and the thing we created
+        // started as pending
```

Comment:
```suggestion
        // We only finalize if we created something and the thing we created
        // started as pending.
```

Comments should usually be full sentences.